engine/chunk/src/main/java/io/deephaven/chunk/WritableCharChunk.java

```diff
@@ -79,6 +79,29 @@ public WritableCharChunk<ATTR> slice(int offset, int capacity) {
         return new WritableCharChunk<>(data, this.offset + offset, capacity);
     }
 
+    // region array
+
+    /**
+     * Get the underlying data array.
+     *
+     * <p>Get the underlying data array.  This Chunk must never be {@link #close() closed} while the array <em>may</em> be in use externally, because it must not be returned to any pool for re-use until that re-use is guaranteed to be exclusive.  This Chunk logically begins at the position indicated by {@link #offset()}.</p>
+     *
+     * @return the underlying data array.
+     */
+    public final char [] array() {
+        return data;
+    }
+
+    /**
+     * Get the offset into the underlying data array.
+     *
+     * @return the offset into the underlying data array.
+     */
+    public final int offset() {
+        return offset;
+    }
+    // endregion array
+
```

Comment:
```suggestion
    // region array
    /**
     * Get the data array backing this WritableCharChunk. The first element of this chunk corresponds to
     * {@code array()[arrayOffset()]}.
     * <p>
     * This WritableCharChunk must never be {@link #close() closed} while the array <em>may</em> be in use externally,
     * because it must not be returned to any pool for re-use until that re-use is guaranteed to be exclusive.
     *
     * @return The backing data array
     */
    public final char[] array() {
        return data;
    }

    /**
     * Get this WritableCharChunk's offset into the backing data array. The first element of this chunk corresponds to
     * {@code array()[arrayOffset()]}.
     *
     * @return The offset into the backing data array
     */
    public final int arrayOffset() {
        return offset;
    }
    // endregion array

```